recipe/meta.yaml

```diff
@@ -57,45 +26,57 @@ build:
 requirements:
   build:
     - python  # [win]
-    - freetype 2.6.*  # [unix]
-    - fontconfig 2.11.*  # [linux]
-    - freeglut  # [linux]
+    - perl
+    - freetype 2.7|2.7.*
+    - fontconfig 2.12.*
+    #- pango  # [linux]
+    #- freeglut  # [linux]
```

Comment:
All the commented out dependencies are those that are not present in @mingwandroid's version. We need to check if we really do not need them. I did notice that his `yum_requirements.txt` installs some of these.